tests/ui/debuginfo/backtrace-dylib-dep.rs

```diff
@@ -7,6 +7,7 @@
 //@ compile-flags: -g -Copt-level=0 -Cstrip=none
 //@ aux-crate: dylib_dep_helper=dylib-dep-helper.rs
 //@ aux-crate: auxiliary=dylib-dep-helper-aux.rs
+//@ ignore-wasm32 spawning processes is not supported
```

Comment:
is there no "needs the ability to spawn processes" directive for compiletest currently, @jieyouxu? wasm32 is not the only platform that has a limited ability to do this.

Reply:
Actually that isn't quite right -- I'm not spawning processes like in `tests/ui/backtrace.rs`, so the previous failure on wasm was for a different reason.

Reply:
I'm not familiar with wasm at all, my guess is maybe this test needs a `//@ needs-unwind` directive instead?

Reply:
Maybe I just need to set `WASMTIME_BACKTRACE_DETAILS=1` here

Reply:
It's because the `wasmtime` test runner doesn't have debug-info enabled, I think?